closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -40,11 +40,16 @@ public class NativeCrypto {
     private static final int ossl_ver = AccessController.doPrivileged(
             (PrivilegedAction<Integer>) () -> {
                 int ossl_ver = -1;
+                String nativeCryptTrace = System.getProperty("jdk.nativeCryptoTrace");
+
                 try {
                     System.loadLibrary("jncrypto"); // check for native library
                     // load OpenSSL crypto library dynamically.
-                    ossl_ver = loadCrypto();
+                    ossl_ver = loadCrypto(((nativeCryptTrace != null) ? 1 : 0));
                 } catch (UnsatisfiedLinkError usle) {
+                    if (nativeCryptTrace != null) {
+                        System.out.println("UnsatisfiedLinkError: Failure attempting to load jncrypto JNI library");
```

Comment:
The native tracing goes to stderr: this should do the same.

Reply:
May I know what do you mean by going to stderr?

Reply:
`System.err.println()` instead of `System.out.println`

Reply:
Edited